src/storage/SplitCache/InLocalStorage.js

```diff
@@ -197,6 +197,18 @@ class SplitCacheLocalStorage {
     log.info('Flushing localStorage');
     localStorage.clear();
   }
+
+  /**
+   * Fetches multiple splits definitions.
+   */
+  fetchMany(splitNames) {
+    const splits = new Map();
+    for (let i = 0; i < splitNames.length; i++) {
```

Comment:
Is there any optimization if using "for (let splitName in splitNames) {" instead of "for (let i = 0; i < splitNames.length; i++) {" ?

Reply:
updated!